coconut/configuration/configuration.go

```diff
@@ -64,6 +67,15 @@ const  (
 	logicError = iota // Logic/Output error
 )
 
+var(
+	white = color.New(color.FgHiWhite).SprintFunc()
+	blue = color.New(color.FgHiBlue).SprintFunc()
+	green = color.New(color.FgHiGreen).SprintFunc()
+	yellow = color.New(color.FgHiYellow).SprintFunc()
+	red = color.New(color.FgHiRed).SprintFunc()
+	grey = color.New(color.FgWhite).SprintFunc()
```

Comment:
Do we use all of these? Are they duplicated with the control utils?

Reply:
Removed unused ones